addons/script.module.pil/addon.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="script.module.pil"
        name="Python Image Library"
-       version="1.1.7"
+       version="3.0.0+matrix.1"
```

Comment:
This will get updated every time Kodi major version is bumped?

Reply:
Will change this and the other one, this is temporary anyway since both are being bumped at the moment